kubernetes/main.tf

```
@@ -0,0 +1,26 @@
+terraform {
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = "4.51.0"
+    }
+  }
+}
+provider "google" {
+  project = "credible-market-426113-k7"
+  zone = "europe-west4-a"
+}
```

We can use some variables here to make it more configurable:

```suggestion
variable "project_id" {
  description = "GCP project ID"
  type        = string
}
variable "zone" {
  description = "GCP zone"
  type        = string
  default     = "europe-west4-a"
}

provider "google" {
  project = var.project_id
  zone = var.zone
}
```